site/content/blog/2020-12-01-whats-new-in-svelte-december-2020.md

```diff
@@ -0,0 +1,65 @@
+---
+title: What's new in Svelte: December 2020
+description: TBD
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+It's the last "What's new in Svelte" of the year... To round out the season, this month's coverage includes _things_!
+
+## New features & impactful bug fixes
```

Comment:
it's not released yet but https://github.com/sveltejs/svelte/pull/5584 is probably the biggest feature. sourcemaps for preprocessor means that sourcemaps will now work with typescript and other languages

Reply:
@d3sandoval do we want to add something about #5584?

Reply:
Doesn't number 5 on my list cover that?

Reply:
Number 5 looks like it's about export maps, which doesn't sound related at all.

#5584 let you use source maps with TypeScript, Sass, Less, etc. in Svelte

Reply:
I just tested with the latest release on the svelte template converted to TS and I still have this offset error. I think it's a huge feature but I feel it's not quite ready yet and needs some more polish. I suggest to move the official announcement to the next monthly edition.

Reply:
Ah, right. I forgot about the offset error (https://github.com/sveltejs/svelte/pull/5584#issuecomment-733811305). @milahu had been looking at that one too and had some ideas, but it probably won't be fixed before the newsletter goes out, so saving for next month sounds reasonable.